src/components/AssignmentTexter/StyleControls.js

```diff
@@ -197,7 +211,7 @@ export const flexStyles = StyleSheet.create({
     // height:105: webkit needs constraint on height sometimes
     //   during the inflection point of showing the shortcut-buttons
     //   without the height, the exit buttons get pushed down oddly
-    height: "15px", //TODO
+    // height: "15px", //TODO
```

Comment:
Is there a reason this line of code is commented out instead of deleted?

Reply:
Good catch Kathy! I don't remember commenting this out, but yes it should be deleted.